x-pack/plugins/security_solution/public/management/components/policies_selector/policies_selector.tsx

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback, useMemo, useState, useEffect, ChangeEvent } from 'react';
+
+import { i18n } from '@kbn/i18n';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFilterGroup,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiFieldSearch,
+  EuiFilterButton,
+  EuiFilterSelectItem,
+  FilterChecked,
+  EuiText,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { ImmutableArray, PolicyData } from '../../../../common/endpoint/types';
+
+export interface PoliciesSelectorProps {
+  policies: ImmutableArray<PolicyData>;
+  defaultIncludedPolicies?: string;
+  defaultExcludedPolicies?: string;
+  onChangeSelection: (items: Item[]) => void;
+}
+
+export interface Item {
```

Comment:
Naming suggestion: `PolicySelectionItem` (just because `Item` is so generic)

Reply:
Totally agree, this was cp from eui docs